consensus/polybft/bridge_manager.go

```diff
@@ -0,0 +1,488 @@
+		txrelayer.WithIPAddress(runtimeConfig.GenesisConfig.Bridge.JSONRPCEndpoint),
+		txrelayer.WithWriter(logger.StandardWriter(&hclog.StandardLoggerOptions{})))
+	if err != nil {
+		return err
+	}
+
+	b.checkpointManager = newCheckpointManager(
+		wallet.NewEcdsaSigner(runtimeConfig.Key),
+		runtimeConfig.GenesisConfig.Bridge.CheckpointManagerAddr,
+		txRelayer,
+		runtimeConfig.blockchain,
+		runtimeConfig.polybftBackend,
+		logger.Named("checkpoint_manager"),
+		runtimeConfig.State)
+
+	eventProvider.Subscribe(b.checkpointManager)
+
+	return nil
+}
+
+// initStateSyncRelayer initializes state sync relayer
+// if not enabled, then a dummy state sync relayer will be used
+func (b *bridgeManager) initStateSyncRelayer(
+	bridgeBackend BridgeBackend,
+	eventProvider *EventProvider,
+	runtimeConfig *runtimeConfig,
+	logger hclog.Logger) error {
+	if runtimeConfig.consensusConfig.IsRelayer {
+		txRelayer, err := getBridgeTxRelayer(runtimeConfig.consensusConfig.RPCEndpoint, logger)
+		if err != nil {
+			return err
+		}
+
+		b.stateSyncRelayer = newStateSyncRelayer(
+			txRelayer,
+			runtimeConfig.State.StateSyncStore,
+			bridgeBackend,
+			runtimeConfig.blockchain,
+			wallet.NewEcdsaSigner(runtimeConfig.Key),
+			&relayerConfig{
+				maxBlocksToWaitForResend: defaultMaxBlocksToWaitForResend,
+				maxAttemptsToSend:        defaultMaxAttemptsToSend,
+				maxEventsPerBatch:        defaultMaxEventsPerBatch,
+				eventExecutionAddr:       contracts.StateReceiverContract,
+			},
+			logger.Named("state_sync_relayer"))
+	} else {
+		b.stateSyncRelayer = &dummyStateSyncRelayer{}
+	}
+
+	eventProvider.Subscribe(b.stateSyncRelayer)
+
+	return b.stateSyncRelayer.Init()
+}
+
+// initStateSyncRelayer initializes exit event relayer
+// if not enabled, then a dummy exit event relayer will be used
+func (b *bridgeManager) initExitRelayer(
+	bridgeBackend BridgeBackend,
+	runtimeConfig *runtimeConfig,
+	logger hclog.Logger) error {
+	if runtimeConfig.consensusConfig.IsRelayer {
+		txRelayer, err := getBridgeTxRelayer(runtimeConfig.GenesisConfig.Bridge.JSONRPCEndpoint, logger)
+		if err != nil {
+			return err
+		}
+
+		b.exitEventRelayer = newExitRelayer(
+			txRelayer,
+			wallet.NewEcdsaSigner(runtimeConfig.Key),
+			bridgeBackend,
+			runtimeConfig.blockchain,
+			runtimeConfig.State.ExitStore,
+			&relayerConfig{
+				maxBlocksToWaitForResend: defaultMaxBlocksToWaitForResend,
+				maxAttemptsToSend:        defaultMaxAttemptsToSend,
+				maxEventsPerBatch:        defaultMaxEventsPerBatch,
+				eventExecutionAddr:       runtimeConfig.GenesisConfig.Bridge.ExitHelperAddr,
+			},
+			logger.Named("exit_relayer"))
+	} else {
+		b.exitEventRelayer = &dummyExitRelayer{}
+	}
+
+	return b.exitEventRelayer.Init()
+}
+
+// initTracker starts a new event tracker (to receive new state sync events)
+func (b *bridgeManager) initTracker(
+	runtimeConfig *runtimeConfig) error {
+	store, err := store.NewBoltDBEventTrackerStore(path.Join(runtimeConfig.DataDir, "/bridge.db"))
```

Comment:
No need for line wrapping, find it slightly easier to read this way
```suggestion
func (b *bridgeManager) initTracker(runtimeConfig *runtimeConfig) error {
```

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/47/commits/64db03d69359faf64096b84b14e11291939e97d4